app/views/hearings/show.html.haml

```diff
@@ -0,0 +1,75 @@
+= govuk_page_title(@hearing_day.strftime('%d/%m/%Y'), t('generic.hearing'))
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Hearing'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Hearing type
+      %td.govuk-table__cell
+        = @hearing.hearing_type
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Court
+      %td.govuk-table__cell
+        = @hearing.court_name || 'not set'
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Time listed
+      %td.govuk-table__cell
+        = @hearing_day&.strftime('%H:%M')
+
+%br
+.govuk-heading-m
+  = 'Attendees'
+
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = 'Attendees'
+  %tbody.govuk-table__body
+    %tr.govuk-table__row
+      %td.govuk-table__cell{ class: 'govuk-!-font-weight-bold' }
+        Defendants
```

Comment:
Inconsistent how you have handled other string using single quotes, plus these should be translated.

@jsugarman

Reply:
all done now